modules/ROOT/pages/deployment/services/s-list/webfinger.adoc

```diff
@@ -0,0 +1,144 @@
+= Webfinger Service Configuration
+:toc: right
+:webfinger-url: https://en.wikipedia.org/wiki/WebFinger
+
+:ext_name: webfinger
+
+// remember to REMOVE the corresponding tab if a new ocis release will be published
+
+:no_second_tab: true
+:no_third_tab: true
+
+:description: The Infinite Scale webfinger service provides an RFC7033 WebFinger lookup of ownCloud instances relevant for a given user account.
+
+== Introduction
+
+{description} {webfinger-url}[WebFinger] is a protocol that allows for discovery of information about people and things identified by an URI.
+
+== OpenID Connect Discovery
+
+Clients can make an unauthenticated `GET \https://drive.ocis.test/.well-known/webfinger?resource=https%3A%2F%2Fcloud.ocis.test` request to discover the OpenID Connect Issuer in the `http://openid.net/specs/connect/1.0/issuer` relation:
+
+[source,json]
+----
+{
+    "subject": "acct:einstein@drive.ocis.test",
+    "links": [
+        {
+            "rel": "http://openid.net/specs/connect/1.0/issuer",
+            "href": "https://sso.example.org/cas/oidc/"
+        }
+    ]
+}
+----
+
+Here, the `resource` takes the instance domain URI, but an `acct:` URI works as well. 
+
+== Authenticated Instance Discovery
+
+When using OpenID connect to authenticate requests, clients can look up the owncloud instances a user has access to.
+
+*   Authentication is necessary to prevent leaking information about existing users.
+*   Basic auth is _not_ supported.
+
+The default configuration will simply return the `OCIS_URL` and direct clients to that domain:
+
+[source,json]
+----
+{
+    "subject": "acct:einstein@drive.ocis.test",
+    "links": [
+        {
+            "rel": "http://openid.net/specs/connect/1.0/issuer",
+            "href": "https://sso.example.org/cas/oidc/"
+        },
+        {
+            "rel": "http://webfinger.owncloud/rel/server-instance",
+            "href": "https://abc.drive.example.org",
+            "titles": {
+                "en": "oCIS Instance"
+            }
+        }
+    ]
+}
+----
+
+== Configure Different Instances Based on OpenidConnect UserInfo Claims
```

Comment:
```suggestion
== Configure Different Instances Based on OpenID Connect User Info Claims
```

Reply:
Do not separate `User Info`. it is  a claim that is really named `UserInfo`